src/gluonts/model/tft/_estimator.py

```diff
@@ -40,13 +40,23 @@
     SetField,
 )
 
-from ._transform import TFTInstanceSplitter
+from ._transform import (
+    BroadcastTo,
+    TFTInstanceSplitter,
+)
 from ._network import (
     TemporalFusionTransformerTrainingNetwork,
     TemporalFusionTransformerPredictionNetwork,
 )
 
 
+def _default_feat_args(dims_or_cardinalities: List[int]):
+    if dims_or_cardinalities:
+        return dims_or_cardinalities
+    return [1]
+
+
+
 class TemporalFusionTransformerEstimator(GluonEstimator):
     @validated()
     def __init__(
```

Comment:
Why is `context_length` required and `prediction_length` optional? Shouldn't it be the opposite?

Reply:
You are right. It should be opposite.